.github/workflows/coverity-scan-fixes.yml

```diff
@@ -3,11 +3,7 @@ name: Coverity Scan master branch on a weekly basis
 on:
   workflow_dispatch:
   schedule:
-    - cron: "0 0,12 * * *"
```

Comment:
What is the coverity change for?

Reply:
@embhorn and I talked about this offline, that it makes sense to run it only once a day. Also fixed the comment to reflect it